ui/app/components/auth-jwt.js

```diff
@@ -109,7 +109,11 @@ export default Component.extend({
   },
 
   exchangeOIDC: task(function*(event, oidcWindow) {
-    let oidcState = event.storageArea.getItem('oidcState');
+    // in non-incognito mode we need to use a timeout because it takes time before oidcState is written to local storage.
+    let oidcState = Ember.testing
```

Comment:
`oidcState` can actually be a `const` since it never gets redefined 😎

Reply:
Good point.  @noelledaley because this is a backport PR, should I do another PR to master with this change and then try and backport that?

Reply:
@Monkeychip yep, that would be great! thank you!

Reply:
One thing to note, it was originally a `let` before this change.  So I suspect it's OK for this PR to go through as is and if we think it's worth it, make another PR to change it to a `const`.  Not ideal, but not sure the best approach 🤷‍♀️

Reply:
good point. perhaps a better idea would be to come back to it the next time we're making changes in this file and not worry about the backport since it's not a breaking change.